example/jwhois.conf

```diff
@@ -56,6 +56,11 @@ whois-servers {
 	"^!?NET\\(BLK\\)?-VRIO\\(-[A-Z0-9]+\\)+$" = "rwhois.verio.net";
 
 	#
+	# Catch ntt.net netblock records
+	#
+	"^!?NET\\(BLK\\)?-VRIO\\(-[A-Z0-9]+\\)+$" = "rwhois.gin.ntt.net";
```

Comment:
This seems as if it would match the same as the previous verio regexp. Are there any examples of an ntt netblock record? The regexp needs to be updated accordingly.